PontoFacil/PontoFacil/Models/ClockIn.cs

```diff
@@ -5,47 +5,112 @@ namespace PontoFacil.Models
 {
     public class ClockIn : BindableBase
     {
+        #region
+        private DateTime? _id;
+        public DateTime? Id
+        {
+            get { return _id; }
+            set { _id = value; }
+        }
+
         private DateTime _start;
         public DateTime Start
         {
             get { return _start; }
-            set { _start = value; }
+            set { SetProperty(ref _start, value); }
         }
 
         private DateTime _end;
         public DateTime End
         {
             get { return _end; }
-            set { _end = value; }
+            set { SetProperty(ref _end, value); }
         }
 
         private int _lunchTime;
         public int LunchTime
         {
             get { return _lunchTime; }
-            set { _lunchTime = value; }
+            set { SetProperty(ref _lunchTime, value); }
+        }
+
+        private DateTime _startLunchTime;
+        public DateTime StartLunchTime
+        {
+            get { return _startLunchTime; }
+            set { SetProperty(ref _startLunchTime, value); }
+        }
+
+        private DateTime _endLunchTime;
+        public DateTime EndLunchTime
+        {
+            get { return _endLunchTime; }
+            set { SetProperty(ref _endLunchTime, value); }
+        }
+
+        private TimeSpan _workedHours;
+        public TimeSpan WorkedHours
+        {
+            get { return _workedHours; }
+            set { SetProperty(ref _workedHours, value); }
+        }
+
+        private TimeSpan _overtimeHours;
+        public TimeSpan OvertimeHours
+        {
+            get { return _overtimeHours; }
+            set { SetProperty(ref _overtimeHours, value); }
+        }
+
+        private TimeSpan _regularHours;
+        public TimeSpan RegularHours
+        {
+            get { return _regularHours; }
+            set { SetProperty(ref _regularHours, value); }
         }
 
-        public void Open(DateTime dt)
+        private bool _isWaiver;
+        public bool IsWaiver
         {
-            _start = dt;
+            get { return _isWaiver; }
+            set { SetProperty(ref _isWaiver, value); }
+        }
+        #endregion
+
+        #region Constructor
+        public ClockIn()
+        {
+            RegularHours = new TimeSpan(8, 0, 0);
+        }
+        #endregion
+
+        #region Methods
+        public void Open(DateTime dt, int lunchTime)
+        {
+            Start = Convert.ToDateTime(dt.ToString("HH:mm:ss"));
+            LunchTime = lunchTime;
+            StartLunchTime = Convert.ToDateTime("12:00:00");
+            EndLunchTime = lunchTime == 1 ? Convert.ToDateTime("13:00:00") : Convert.ToDateTime("14:00:00");
```

Comment:
Could you extract the `Convert.ToDateTime` in a global var named `ONE_PM` and `TWO_PM` ou something like it and extract the operation to a method. The reason is make you code more readable.